Meta/CMake/vcpkg/overlay-ports/angle/cmake-buildsystem/CMakeLists.txt

```diff
@@ -257,6 +257,9 @@ if (USE_ANGLE_EGL OR ENABLE_WEBGL)
     )
     target_compile_definitions(ANGLE PUBLIC ${ANGLE_DEFINITIONS})
     target_link_libraries(ANGLE PRIVATE ZLIB::ZLIB ${CMAKE_DL_LIBS})
+    if (CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
+        target_link_libraries(ANGLE PUBLIC pthread)
```

Comment:
the "correct" way to do this would be to find_package(Threads) and then always link against Threads::Threads, regardless of platform. but this hack is fine.